components/Header/src/index.scss

```diff
@@ -23,17 +22,18 @@
 .denhaag-header__container {
   margin-inline-end: auto;
   margin-inline-start: auto;
-  padding-inline-end: var(--denhaag-header-grid-gutter);
-  padding-inline-start: var(--denhaag-header-grid-gutter);
-  width: min(var(--denhaag-header-max-width), 100%);
-  max-width: var(--denhaag-header-max-width);
+  padding-inline: var(--denhaag-header-grid-gutter, var(--denhaag-space-xs));
+  width: min(var(--denhaag-header-max-width, 100%), 100%);
+  max-width: var(--denhaag-header-max-width, 100%);
```

Comment:
Is deze max-width wel nodig? De width wordt al de kleine variant van de 2 genomen..

Reply:
Ik dacht van niet. Eigenlijk laten staan, omdat ik dacht er waarschijnlijk een extra rede was dat deze gezet werd.